components/fxa-client/src/fxa_client.udl

```diff
@@ -886,6 +891,13 @@ dictionary SendTabPayload {
   string stream_id;
 };
 
+// ...
+//
+dictionary CloseTabsPayload {
+  // ...
+  sequence<string> urls;
```

Comment:
We talked for a bit about changing the payload to only close one tab at a time, and I wanted to capture that conversation here!

Our rationale for supporting just one tab is that we'll still need to:

* Raise the rate limit for FxA's `/v1/account/devices/invoke_command` endpoint, and...
* Collapse pushes that FxA sends to the receiving device's `pushCallback`, and...
* Coalesce the user-visible "N tabs closed" notifications on the receiving device.

...Regardless of whether we include one URL or multiple URLs in the payload.

Including multiple URLs means we need to batch closed tabs client-side, _and_ still handle the case where a batch might need to be split into multiple commands—if the user closes lots of tabs at once, or waits longer than the "undo" delay to close more, for example.

IOW, supporting multiple URLs complicates our implementation a bit, but doesn't avoid having to do the three things above.

Reply:
Yeah, while that's true, there's also no real harm in keeping this a sequence, right? (ie, I'm not sure if you are suggesting we should drop the sequence here or not - I'd say "no" just as risk mitigation, but could be convinced otherwise)

Also wonder if "Payload" is the correct term here? Payload implies "over the wire" but we would be free to transform the "payload" into this "client request". If it wasn't so verbose `IncomingCloseTabsRequest` captures everything the reader needs to know (ie, it's not used for outgoing close requests etc)

[eek, just noticed `Payload` is already used for SendTab, so what you wrote is consistent, so feel free to completely ignore the above :)]

Reply:
Closing the loop on this:

1. Yep, let's keep this a sequence, so that we can have the best of both worlds.
2. I like `Request` a lot, but I agree; I think we'd want to change Send Tab to use the same nomenclature. Maybe a good first bug for someone? 😉 I think the monorepofication will make these kinds of renames a lot easier, too.